DiscountDiscover/AppDelegate.m

```diff
@@ -20,6 +21,9 @@ @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     
+    // start updating location
+    [LocationManager sharedLocationManager];
```

Comment:
Would be better to make this explicit instead of a side effect of the `init` method, like `[[LocationManager sharedLocationManager] startMonitoring];`.